src/vid2805-dual-stepper-element.ts

```diff
@@ -0,0 +1,191 @@
+import { html, LitElement, svg } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ElementPin } from '.';
+import { mmToPix } from './utils/units';
+
+export type HandShape = 'arrow' | 'plain'; // | 'fancy';
+
+export class StepperHand {
+  /**
+   * length used to control hand length
+   */
+  length = 25;
+
+  /**
+   * colour used to paint the hand
+   */
+  colour = '';
+
+  /**
+   * hand shape selects from predefined shape paths
+   */
+  shape: HandShape = 'plain';
+
+  /**
+   * the current hand angle
+   */
+  angle = 0;
+
+  constructor(length = 25, colour = '', shape: HandShape = 'plain', angle = 0) {
+    this.angle = angle;
+    this.colour = colour;
+    this.shape = shape;
+    this.length = length;
+  }
+}
+
+@customElement('wokwi-vid2805-dual-stepper')
+export class VID2805DualStepperElement extends LitElement {
+  @property() outerHand = new StepperHand();
```

Comment:
Encapsulating the hand properties in an object is neater, but I'm afraid it causes two issues:

1. It makes the element unusable from HTML. You always have to use JavaScript to create the `innerHand`/`outerHand` objects.
2. lit is unable to track the properties inside `innerHand`/`outerHand`. Thus, if you change any of the properties (e.g. `document.querySelector('wokwi-vid2805-dual-stepper').innerHand.angle = 40`), it won't pick up the change. The only way is to create a new `innerHand` object every time you want to change a property.

So I think that in this situation and given the limitations of the technology (lit + html), we should change the design to have individual properties per each hand attribute (`innerAngle`, `innerColor`, etc). This will also fix the issue with the story controls not working.

Reply:
as discussed, removed